src/CredentialRepository.php

```diff
@@ -228,7 +228,7 @@ public function serialize()
      * and the PHP version used in less than PHP 9.0
      *
      * @param string $serialized
-     * @deprecated will be removed in 5.0
+     * @deprecated 4.5.0 Use __unserialize() insteaed
```

Comment:
No corresponding deprecation notice?
```suggestion
     * @deprecated 4.5.0 Use __unserialize() instead
```

Reply:
Updated